SphericalShell/OpenMC_Output_Processing/SS_all_PostProcessing.py

```diff
@@ -0,0 +1,47 @@
+import openmc
+import openmc.deplete
+import yaml
+import argparse
+from SS_Tally_PostProcessing import extract_tally_values, plot_flux_tally, save_data
+from SS_Depletion_PostProcessing import extract_nuclides, plot_save_data
+from OpenMC_SS_Material import alara_element_densities, make_element
+
+def main() : 
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--yaml_postprocess_path', default = 'SS_Post_Processing_YAML.yaml', help="Path to YAML file containing inputs for post-processing (str)")
+    parser.add_argument('--yaml_model_path', default = 'OpenMC_SS_YAML.yaml', help="Path to YAML file containing required inputs to build model (str)")
+    args = parser.parse_args()
+    yaml_postprocess_path = args.yaml_postprocess_path
+    yaml_model_path = args.yaml_model_path
+    
+    with open(yaml_postprocess_path, 'r') as yaml_pp :
+        pp_inputs = yaml.safe_load(yaml_pp)
+    indices = pp_inputs['indices']
+    filepaths = pp_inputs['filepaths']
+    units = pp_inputs['units']
+    
+    with open(yaml_model_path, 'r') as model_file:
+        model_inputs = yaml.safe_load(model_file)
+    elelib_fp = model_inputs['elelib_fp']    
+    element = model_inputs['element']
+    inner_radius = model_inputs['geom_info']['inner_radius']
+    thickness = model_inputs['geom_info']['thickness'] 
+    
+    # Tally post-processing:
+    etv = extract_tally_values(filepaths['statepoint_file_path'])

```

Comment:
If you return multiple values with different intents, it's best to separate them ASAP.  One way that should work is:
```suggestion
    tallies, tally_array = extract_tally_values(filepaths['statepoint_file_path'])
```